palettes/palettes.json

```diff
@@ -8097,6 +8097,42 @@
     "#ffcb8b",
     "#ffffff"
   ],
+  "Oxocarbon Dark": [
+    "161616", 
+    "262626", 
+    "393939", 
+    "525252", 
+    "dde1e6", 
+    "f2f4f8", 
+    "ffffff", 
+    "08bdba",
+    "3ddbd9", 
+    "78a9ff", 
+    "ee5396", 
+    "33b1ff", 
+    "ff7eb6", 
+    "42be65", 
+    "be95ff", 
+    "82cfff"
+  ],
+  "Oxocarbon Light": [
```

Comment:
Is there an official source for the palette? I found https://github.com/nyoom-engineering/oxocarbon.nvim/blob/main/fnl/oxocarbon/init.fnl#L77-L94 which mostly matched, but seems like it has some differences